css/properties/scroll-snap-type.json

```diff
@@ -9,25 +9,29 @@
               "version_added": false
```

Comment:
This should also be 69.

Reply:
Is this a GitHub bug? Your comment appears in different lines 😮 

![image](https://user-images.githubusercontent.com/3396686/45723457-59ea3f80-bbec-11e8-9bec-68d5f7c8c44a.png)

![image](https://user-images.githubusercontent.com/3396686/45723453-535bc800-bbec-11e8-89d4-576fbf29b241.png)